### ISSUE TITLE
Wacom One Small not recognized

### ISSUE THREAD

**Issue by SreckoM** [linuxwacom/libwacom#22]
On Ubuntu 18.04 Wacom One (CTL-672) is not being recognized by Ubuntu. 

_lsusb | grep Wacom_ gives `Bus 001 Device 006: ID 056a:037b Wacom Co., Ltd`

**Comment by jigpu** [comment]
Adding a reference to https://github.com/linuxwacom/xf86-input-wacom/issues/20 since it may be a similar issue...

**Comment by jigpu** [comment]
Would you mind running [our sysinfo.sh script](https://github.com/linuxwacom/wacom-hid-descriptors/blob/master/scripts/sysinfo.sh) and attaching the generated tarball to this thread? It will contain several key logs that might give a clue about what is going on.

**Comment by whot** [comment]
Closing after months of tumbleweeds

**Comment by tcdude** [comment]
Stumbled upon this while trying to get my Wacom One to work. Attached the requested file. Hope this gets reopened and resolved.
[sysinfo.vpywTfre86.tar.gz](https://github.com/linuxwacom/libwacom/files/2608033/sysinfo.vpywTfre86.tar.gz)

**Comment by jigpu** [comment]
@tcdude the logs in your particular sysinfo output indicate that the driver and kernel versions don't match. Try installing the latest system updates, rebooting, and then deleting and rebuilding input-wacom from source.

**Comment by tcdude** [comment]
@jigpu I did and it worked more or less. The tablet is now recognized, but in the settings panel it doesn't recognize the pen (That was on ubuntu 18.04). After going over the supported DeviceIDs I saw that it supposedly is supported with kernel version 4.16+ so I figured that I give ubuntu 18.10 a try (which ships with 4.17 afaik). No luck, the pen is still not recognized but the tablet is recognized and works (just not as expected/intended). 
I guess I need to open a new issue for my case, where the tablet works but the pen isn't recognized?
